src/assets/icons/tab-bar/contact-tracing-alert.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No svgo on this one?

Reply:
looks like another bunch of svgs where not optimized